Architecture.md

```diff
@@ -12,7 +12,33 @@
 
 #### Web application
 
-***TODO: [CLIENT] fill for web application frontend***
+The website is made using [React](https://react.dev/), we decided to use this framework because it is the most popular, widely used in industry and greatly documented. Furthermore, the whole team was already familiar with React thanks to several past projects.
+
+As for the component library, we chose [Chakra UI](https://chakra-ui.com/). Even though we didn't have any experience with it, we decided to try it out since it was visualy appealing and popular. It is also very well documented and supported by the community.
+
+[Typescript](https://www.typescriptlang.org/) was a must have for us to ensure code quality and to avoid bugs. Using a strongly typed codebase is also a good way to ensure the maintainability of our project in the future.
```

Comment:
```suggestion
[TypeScript](https://www.typescriptlang.org/) was a must-have for us to ensure code quality and to avoid bugs. Using a strongly typed codebase is also a good way to ensure the maintainability of our project in the future.
```